src/Components/Details/Details.tsx

```diff
@@ -3,11 +3,13 @@ import "./Details.css";
 import Review from "../Review/Review";
```

Comment:
This is a really big file. Totally optional, but it would be nice if this was broken down further into components. For example, there's a clear review component in here. There might also be room for a swiper component, etc. 

Ultimately, this is optional because you probably aren't re-using those pieces anywhere else in the codebase. However, some developers find it nice to have things organized into small chunks so that when something gets updated, it's easy to find and is less likely to negatively impact other areas of the code.